### ISSUE TITLE
Some problem about result.csv

### ISSUE THREAD

**Issue by LeeXiaosong** [akanametov/pathgan#4]
hi, akanametov thank you for your contribution to Pathgan.  When i run get_logs.py, I found that there is no result.csv file under the specified path, can you tell me how I should generate this file. More detailed questions I have sent to your email [akkanametov@gmail.com]. Thank you very much.

**Comment by akanametov** [comment]
> hi, akanametov thank you for your contribution to Pathgan. When i run get_logs.py, I found that there is no result.csv file under the specified path, can you tell me how I should generate this file. More detailed questions I have sent to your email [[akkanametov@gmail.com](mailto:akkanametov@gmail.com)]. Thank you very much.

Thanks for interest! Unfortunately, the notebook for generation of that .csv file was not included in repo. However, you can find that .csv file itself with generated ROIs in the links section: https://github.com/akanametov/pathgan#links Links 4-7 are results of the both GANs on generated and movingai datasets. Actually that .csv file just consists of file paths to map, task and ROI with metrics (IS, FID, IoU, DICE). So if you consider generation of that .csv file, you can use IS and FID metrics provided in `pathgan.metrics`, but you need to implement IoU, DICE metrics. Nevertheless, I can adress that issue during the week, create and provide that notebook, if needed.

**Comment by LeeXiaosong** [comment]
> 

I think the result.csv file is generated at the same time in the train.py code about the mapping of roi and map/tasks, but there is no corresponding work in the file you uploaded. Also, in analyzes_of_connectivity.ipynb, the execution of the RRT() code reports an error, and I don't know where this function is.

**Comment by akanametov** [comment]
> > 
> 
> I think the result.csv file is generated at the same time in the train.py code about the mapping of roi and map/tasks, but there is no corresponding work in the file you uploaded. Also, in analyzes_of_connectivity.ipynb, the execution of the RRT() code reports an error, and I don't know where this function is.

Thanks! In that case I will fix this issues during the week or less and provide updated scripts as well as notebooks. The RRT algorithm can be found in `pathgan.models.rrt`

**Comment by LeeXiaosong** [comment]
> > > 
> > 
> > 
> > 我认为在关于roi和map/tasks映射的train.py代码中同时生成了result.csv文件，但是你上传的文件中没有对应的工作。还有，analyses_of_connectivity.ipynb中，执行RRT()代码报错，不知道这个函数在哪里。
> 
> 谢谢！在这种情况下，我将在一周或更短的时间内解决此问题，并提供更新的脚本和笔记本

Thank you very much for your contribution and support. I sent you an email asking for advice on some questions,which have been bothering me for a long time, would it be convenient for you to check the email.

**Comment by akanametov** [comment]
> > > > 
> > > 
> > > 
> > > 我认为在关于roi和map/tasks映射的train.py代码中同时生成了result.csv文件，但是你上传的文件中没有对应的工作。还有，analyses_of_connectivity.ipynb中，执行RRT()代码报错，不知道这个函数在哪里。
> > 
> > 
> > 谢谢！在这种情况下，我将在一周或更短的时间内解决此问题，并提供更新的脚本和笔记本
> 
> Thank you very much for your contribution and support. I sent you an email asking for advice on some questions,which have been bothering me for a long time, would it be convenient for you to check the email.

Could you please send it to me once again. I can't find it in my mail box

**Comment by LeeXiaosong** [comment]
> 

Maybe I need to check if [akkanametov@gmail.com] is your email first?[akkanametov@gmail.com]

**Comment by akanametov** [comment]
Yes, this is it! But for some reason I don't see that particular message

ср, 7 сент. 2022 г. в 10:05, LeeXiaosong ***@***.***>:

**Comment by LeeXiaosong** [comment]
> 

I have sent it again, thanks for checking.

**Comment by HL-ESLAB** [comment]
Hello, Lee, has your problem been solved? I also encountered the same problem when running this project.
